### DISCUSSION TITLE
Github Students Certification cannot use the phone's rear camera to take photos

### DISCUSSION THREAD

**Discussion by PengJiahao7890** [community/community#66461] [GitHub Education]
### Hi everyone,

Github Students Certification cannot use the phone's rear camera to take photos while taking academic status photos.I tried both iPhone and Android, using the edge browser and Chrome, respectively.I hope someone can help me with this problem. Thank you.

**Comment by Messjer** [comment]
Same here. I submitted a bug report to Github but got two template response that were irrelevant to this issue in a row. It was my first time using Github Support and I was really disappointed.
